crates/env/src/api.rs

```diff
@@ -371,7 +371,7 @@ where
 ///
 /// # Note
 ///
-/// - The `filtered_keys` can be used to ignore certain storage regions
+/// - `filtered_keys` can be used to ignore certain storage regions
```

Comment:
@drahnr This is a workaround which I used, since the crate ignores text in `…`. Taking this ignoring-behavior into consideration the sentence would then be "The can be used …" ‒ which results in an error. Is there a better way?